src/component.py

```diff
@@ -9,14 +9,21 @@ class Component(ABC):
     event_bus: EventBus
     priority: int
 
-    def __init__(self, event_bus: EventBus, priority: int):
+    PRIORITY_LEVELS: dict[str, int] = {
```

Comment:
I thought about real enums, actually 😇 We could compare them and use them as types. But if you are happy with this solution, then I am too.

Reply:
I think it is sufficient for now. But we can change that in the future if it is necessary? 🤔